test/nlp/test_autohf.py

```diff
@@ -71,6 +71,12 @@ def test_hf_data():
 
     del automl
 
+    import os
+    import shutil
+
+    if os.path.exists("test/data/output/"):
+        shutil.rmtree("test/data/output/")
```

Comment:
Is it a good idea to add in the documentation to tell users that the output may consume large space on disk and how to clean it etc.? Could create another issue.

Reply:
Added.